### PR TITLE
privatelink updates needed - Clarifications

### PR DESCRIPTION
**Pull request by jpanke**
Needed to clarify what https is (its 443) as well as ensuring that I added what the principle needs to be filled out with for easier access.   arn:aws:iam::402592597372:root

## Purpose of this pull request
This pull request (PR) clarifies the PrivateLink setup by adding port 443 along with explicit instruction for giving access to the Magento account principal to the service endpoint.

## Affected DevDocs pages
src/cloud/project/privatelink-service.md

### CODE REVIEW

File: src/cloud/project/privatelink-service.md [hunk 1]
```diff
@@ -61,6 +62,7 @@ Enabling PrivateLink can take up to 5 business days. Providing incomplete, or in
    -  **Magento Cloud Project ID**–Provide the {{site.data.var.ece}} Pro project ID. You can get the Project ID and other project information using the folllowing [Magento Cloud CLI][] command:  ```magento-cloud project:info```
    -  **Connection type**–Specify unidirectional or bidirectional for connection type
    -  **Service endpoint**–For bidirectional PrivateLink connections, provide the DNS URL for the VPC service endpoint that Magento must connect to, for example `com.amazonaws.vpce.<cloud-region>.vpce-svc-<service-id>`.
+   -  **Service endopoint access granted**-Ensure that your service endpoint has the principle set to allow access to= `arn:aws:iam::402592597372:root`.  If this is not set the bi-direction connection to access the service in your VPC cannot be added and will delay the setup.
```
Comment (bdenham) [inline src/cloud/project/privatelink-service.md:65]:
```suggestion
   -  **Service endpoint access granted**-Provide the Magento account principal with access to your service endpoint: `arn:aws:iam::402592597372:root`. If this access is not provided, the bidirectional PrivateLink connection to the service in your VPC will not be added, which will delay the setup.
```

File: src/cloud/project/privatelink-service.md [hunk 2]
```diff
@@ -89,7 +91,7 @@ The following workflow outlines the enablement process for PrivateLink integrati
 
    -  **Magento** supplies the Magento account principal (root user for AWS or Azure account) and requests access to the customer VPC endpoint service.
 
-   -  **Customer** enables Magento access to the endpoint service in customer VPC.
+   -  **Customer** enables Magento access to the endpoint service in customer VPC.  There is prereq of ensuring principle access= `arn:aws:iam::402592597372:root`
```
Comment (bdenham) [inline src/cloud/project/privatelink-service.md:94]:
```suggestion
   -  **Customer** enables Magento access to the endpoint service in the customer VPC. This assumes that the Magento account principal has access to `arn:aws:iam::402592597372:root`, as previously described in the **Service endpoint access granted** prerequisite.
```

**Review by bdenham — APPROVED** [review]
Check and commit changes.

File: src/cloud/project/privatelink-service.md [hunk 3]
```diff
@@ -25,7 +25,8 @@ The PrivateLink service integration for {{site.data.var.ece}} projects includes
 -  You cannot establish SSH connections using PrivateLink. For SSH, use the Magento SSH capabilities. See [Enable SSH keys][].
 -  Magento support does not cover troubleshooting AWS PrivateLink issues beyond initial enablement.
 -  Customers are responsible for costs associated with managing their own VPC.
--  You cannot use the HTTPS protocol to connect to Magento Commerce over PrivateLink.
+-  You cannot use the HTTPS (443) protocol to connect to Magento Commerce over PrivateLink.
```
Comment (bdenham) [inline src/cloud/project/privatelink-service.md:28]:
```suggestion
-  You cannot use the HTTPS protocol (port 443) to connect to Magento Commerce over PrivateLink.
```

**Review by bdenham — CHANGES_REQUESTED** [review]

**Review by bdenham — APPROVED** [review]

**Comment by jpanke** [comment]
this is good changes.

**Comment by bdenham** [comment]
@magento import code to magento-devdocs/devdocs

**Comment by magento-engcom-team** [comment]
@bdenham the branch with code successfully imported into`magento-devdocs/devdocs` repository. Branch name: `imported-magento-devdocs-8771`.

**Pull request merged**